backend/src/api/vulnerabilities.ts

```diff
@@ -179,13 +179,23 @@ export const update = wrapHandler(async (event) => {
   }
   if (vuln && isAuthorized) {
     const body = JSON.parse(event.body);
+    const user = await User.findOne({
+      id: event.requestContext.authorizer!.id
+    });
     if (body.substate) {
-      const user = await User.findOne({
-        id: event.requestContext.authorizer!.id
-      });
       vuln.setState(body.substate, false, user ? user : null);
     }
     if (body.notes) vuln.notes = body.notes;
+    if (body.comment) {
+      vuln.actions.unshift({
```

Comment:
Looks good -- can you add a test so we can make sure adding a comment is covered?